index.html

```diff
@@ -5,8 +5,33 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/e2597287a1.js" crossorigin="anonymous"></script>
   <title>Portfolio</title>
 </head>
 <body>
+  <div class="app-cont">
+    <div class="header-cont">
+      <div class="nav-cont">
+        <div class="header-img"><img src="images/image_geometry_1.png" alt="">
+        </div>
```

Comment:
- [ ][OPTIONAL] you should change this `<div>` to `<button>`that make sense than div cause it's a button